### DISCUSSION TITLE
PlatformIO IDE non funziona più ?!?

### DISCUSSION THREAD

**Discussion by mippo9** [indomus/forum#2339] [Hardware & Firmware]
Buongiorno, sono nuovo, volevo provare a "flashare" un Sonoff.
Non riesco ad installare PlatformIO IDE su Atom, sembra irraggiungibile il sito ...
Qualcuno ne sa qualcosa ?
Ci sono alternative ?

grazie
mippo

**Comment by snakuzzo** [comment]
Ti sei perso [QUESTO](https://github.blog/2022-06-08-sunsetting-atom/)
Passa a VSCode. Leggi [QUI](https://tasmota.github.io/docs/Visual-Studio-Code/)

**Reply by KC-inDomus** [reply]
Diego Atom funziona perfettamente. hanno solo revocato quelle versioni indicate in quell'articolo.
